ocs_ci/deployment/acm.py

```diff
@@ -157,6 +158,28 @@ def download_binary(self):
                 os.path.expanduser("~/.local/bin/subctl"),
                 os.path.join(config.RUN["bin_dir"], "subctl"),
             )
+        elif self.source == "downstream":
+            self.download_downstream_binary
+
+    def download_downstream_binary(self):
+        subctl_ver = config.ENV_DATA["subctl_version"]
+        processor = platform.processor()
+        if processor == "x86_64":
+            binary_pltfrm = "amd64"
+        cmd = (
+            f"oc image {constants.SUBCTL_DOWNSTREAM_URL}{subctl_ver} "
```

Comment:
missing extract option in the cmd

Reply:
We will need to use:
--filter-by-os 
With value
linux/arm64 - Apple M chips
linux/amd64   - regular x86_64


Some example from local testing:
```Console
$ oc image extract --filter-by-os linux/arm64 --registry-config ./path-to/pull-secret-ocs-ci registry.redhat.io/rhacm2/subctl-rhel8:v0.14 --confirm --path "/dist/subctl-v0.14*-linux-*.tar.xz":./
$ ls -l
total 43752
-rw-r-----@ 1 pbalogh  staff    11M Nov 16 00:32 subctl-v0.14.7-linux-amd64.tar.xz
-rw-r-----@ 1 pbalogh  staff   9.9M Nov 16 00:31 subctl-v0.14.7-linux-arm64.tar.xz
```

The previous file was `subctl-v0.14.7-linux-amd64.tar.xz` was actually downloaded before when I tried without specifying the `--filter-by-os linux/arm64 `

Reply:
done